mobile/assets/i18n/de-DE.json

```diff
@@ -1,8 +1,8 @@
 {
-  "action_common_back": "Back",
+  "action_common_back": "Zurpck",
```

Comment:
```suggestion
  "action_common_back": "Zurück",
```

Reply:
I've seen that and I'll fix it in localizely anyways, but thanks! :)